### PR TITLE
[WIP, ENH, STY, REF] Refactor seedVoxelCorrelation.sh

### PR DESCRIPTION
**Pull request by tbweng**
Changes proposed in this pull request
- enhanced arg parsing
- add flag for producing seedmaps (default behavior to extract timeseries only)
- if producing seedmaps, remove highres mapping
- if motionscrubbing, do both unscrubbed and scrubbed (no 0, 1, or 2 option like previously used).

Todo:
- make separate rois (rois_classic vs rois_compcor?) directory for distinguishing between classic and compcor roi timeseries.

Successful tests:
```
git checkout enh_seedVoxelCorrelation.sh
bash seedVoxelCorrelation.sh \
--epi=/Shared/vosslabhpc/Projects/Bike_ATrain/Imaging/BIDS/derivatives/test/sub-GEA161_ses-activepre/ica_aroma/denoised_func_data_nonaggr_bp_res4d_normandscaled.nii.gz \
--roiList=/Shared/vosslabhpc/Projects/Bike_ATrain/Imaging/BIDS/derivatives/seeds/seedlist.txt \
--compcor

bash seedVoxelCorrelation.sh \
--epi=/Shared/vosslabhpc/Projects/Bike_ATrain/Imaging/BIDS/derivatives/test/sub-GEA161_ses-activepre/ica_aroma/denoised_func_data_nonaggr_bp_res4d_normandscaled.nii.gz \
--roiList=/Shared/vosslabhpc/Projects/Bike_ATrain/Imaging/BIDS/derivatives/seeds/seedlist.txt \
--compcor --seedmaps
```

### CODE REVIEW

File: seedVoxelCorrelation.sh [hunk 1]
```diff
@@ -19,18 +19,13 @@ VossLabMount="$(mount | grep vosslabhpc | awk '{print $3}')"
 function Usage {
   echo "Usage: seedVoxelCorrelation.sh -E restingStateImage -r roi -m motionscrubFlag -n -f"
```
Comment (mwvoss) [inline seedVoxelCorrelation.sh]: usage example still seems to have old parsing?

File: seedVoxelCorrelation.sh [hunk 2]
```diff
@@ -19,18 +19,13 @@ VossLabMount="$(mount | grep vosslabhpc | awk '{print $3}')"
...
-  echo "        *e.g. -r pccrsp -r icalc"
-  echo "   -R Data file with seed list, one seed per line"
+  echo "   --motionscrub use motionscrubbed and non-motionscrubbed EPI (parallel output)"
+  echo "   --roiList Data file with seed list, one seed per line"
   echo "        **Use ONLY one option, -r or -R, NOT both"
-  echo "   -n Nuisance regression feat directory"
-  echo "        e.g., nuisancereg_compcor.feat"
+  echo "   --compcor"
+  echo "   --seedmaps print seedmaps (default is off)"
   echo "   -f (fieldMap registration correction)"
```
Comment (mwvoss) [inline seedVoxelCorrelation.sh]: if -f for other scripts that's fine, but consider -fmap to be easier to interpret from call

File: seedVoxelCorrelation.sh (hunk 1 above)
Comment (tbweng) [reply seedVoxelCorrelation.sh]: Yeah, I realized that after the initial commit+push so I tried to fix then <git commit --amend> but it resulted in a separate commit: https://github.com/HBClab/RestingState/pull/74/commits/184f190e96a70b43ed137516eca08dbd0325793f

**Comment by mwvoss** [comment]
all latest edits look ok to me

**Pull request merged**